src/main/webapp/createEvent.html

```diff
@@ -48,16 +48,16 @@ <h1 class=center>Create Your Event :)!</h1>
                   <input type="text" name="state" placeholder="Enter valid state" > 
                         
                   <label for="zip"> Zip </label>
-                  <input type="text" name="zip" placeholder="Enter valid zip"> 
+                  <input type="number" name="zip" placeholder="Enter valid zip"> 
```

Comment:
Will changing it to a type of number allow for leading zeros?

Reply:
The input's value is always a string (at least in all of our inputs). The attribute "type" only limits the characters allowed and the format. The leading zero will not go away.